conda_smithy/cli.py

```diff
@@ -408,6 +408,63 @@ def __call__(self, args):
         sys.exit(int(not all_good))
 
 
+POST_SKELETON_MESSAGE = """
+A skeleton for using CI has been generated! Please use the following steps
+to complete the CI process:
+
+1. Fill out {args.recipe_directory}/meta.yaml with your install and test code
+2. Commit all changes to the repo.
+
+        $ git add . && git commit -am "ran conda smithy skeleton"
+
+3. Remember to register your repo with the CI providers.
+4. Rerender this repo to generate the CI configurations files.
+   This can be done with:
+
+        $ conda smithy rerender -c auto
+
+At any time in the future, you will be able to automatically update your CI configuration by
+re-running the rerender command above. Happy testing!
+"""
+
+
+class CISkeleton(Subcommand):
+    subcommand = "ci-skeleton"
+
+    def __init__(self, parser):
+        super(CISkeleton, self).__init__(
+            parser, "Generate skeleton for using CI outside of a feedstock"
+        )
+        scp = self.subcommand_parser
+        scp.add_argument(
+            "--feedstock-directory",
+            default=os.getcwd(),
+            help="The directory of the feedstock git repository.",
+            dest="feedstock_directory",
+        )
+        scp.add_argument(
+            "-r",
+            "--recipe-directory",
+            default="recipe",
+            dest="recipe_directory",
+        )
+        scp.add_argument("package_name", nargs="?")
+
+    def __call__(self, args):
+        from conda_smithy.ci_skeleton import generate
+
+        # complete configuration
+        if not hasattr(args, "package_name"):
```

Comment:
Even with `nargs="?"` this attribute will exist, right? At least I think so; it's been a while that I did things with `argparse`...
This should work in any case:
```suggestion
        if getattr(args, "package_name") is None:
```